docs/source/data.rst

```diff
@@ -88,34 +88,27 @@ will provide the most information.
 The export files of the following search engines have been tested for their
 acceptance in ASReview:
 
-+-----------------+----------+----------+----------+-----------+
-|                 | **.ris** | **.tsv** | **.csv** |  **.xlsx**|
-|                 |          |          |          |           |
-+-----------------+----------+----------+----------+-----------+
-|**CINHAL**       | X        | N/A      | X        | N/A       |
-|**(EBSCO)**      |          |          |          |           |
-+-----------------+----------+----------+----------+-----------+
-|**Cochrane**     | ✅       | N/A      | ✅       | N/A       |
-+-----------------+----------+----------+----------+-----------+
-| **Embase**      | ✅       | N/A      | ✅       | ✅        |
-+-----------------+----------+----------+----------+-----------+
-|**Eric (Ovid)**  | X        | N/A      | N/A      | X         |
-+-----------------+----------+----------+----------+-----------+
-|**Psychinfo**    | X        | N/A      | N/A      | X         |
-|**(Ovid)**       |          |          |          |           |
-+-----------------+----------+----------+----------+-----------+
-| **Pubmed**      | X        | N/A      | X        | N/A       |
-+-----------------+----------+----------+----------+-----------+
-| **Scopus**      | ✅       | N/A      | ✅       | N/A       |
-+-----------------+----------+----------+----------+-----------+
-|**Web of**       | X        | X        | N/A      | N/A       |
-|**Science**      |          |          |          |           |
-+-----------------+----------+----------+----------+-----------+
+==================== ======== ======== ======== =========
+\                    **.ris** **.tsv** **.csv** **.xlsx**
+==================== ======== ======== ======== =========
+**CINAHL (EBSCO**)   ✅       N/A      ☑️       N/A
+**Cochrane**         ✅       N/A      ✅       N/A
+**Embase**           ✅       N/A      ✅       ✅
+**Eric (Ovid)**      ✅*      N/A      N/A      N/A
+**Psychinfo (Ovid)** ✅*      N/A      N/A      N/A
+**Pubmed**           X        N/A      X        N/A
+**Scopus**           ✅       N/A      ✅       N/A
+**Web of Science**   ✅       N/A      N/A      N/A
+==================== ======== ======== ======== =========
 
 -  ✅ = The data can be exported from the search engine and imported in ASReview.
+-  ☑️ = The data can be exported from the search engine and imported in ASReview after
+   adjusting column names (see :ref:`data_format:Data format`).
```

Comment:
Can you provide an example? I prefer to fix the issue instead of documenting this. It can be added to https://github.com/asreview/asreview/blob/6d2e12549aa9708311c7b7bc661866bc506c4115/asreview/config.py#L48-L65.

Reply:
The column name in the csv file is `Article Title`
![image](https://user-images.githubusercontent.com/28191416/184634829-f78153cd-e64d-4ba7-aeeb-d97d59e6db37.png)

Reply:
Oke thanks!